server/services/caseloadService.ts

```diff
@@ -73,47 +72,6 @@ export default class CaseloadService {
       .then(caseload => this.mapResponsibleComsToCases(caseload))
   }
 
-  async getOmuCaseload(user: User, prisonCaseload: string[]): Promise<OmuCaselist> {
```

Comment:
Assume this and the `pairNomisRecordsWithDelius` are no longer being used or have already been extracted out to somewhere else? 😅 sorry if I've missed this

Reply:
Maybe these are included as the static helpers you refer to in your comment above actually 😅